model/metrics/dotnet/dotnet-health_checks.yaml

```
@@ -0,0 +1,49 @@
+groups:
+  - id: health_checks
+    prefix: health_checks
+    type: attribute_group
+    brief: Health Check attributes
+    attributes:
+      - id: dotnet.health_check.status
```

```suggestion
      - id: health_check.status
```
If you use the proposed `dotnet` prefix in the attribute group, then here you don't need to repeat it _I think_